tests/replication2_ssl.erl

```diff
@@ -253,6 +273,12 @@ confirm() ->
     ?assertEqual(ok, test_connection({Node1, merge_config(SSLConfig5, BaseConf)},
             {Node2, merge_config(SSLConfig6, BaseConf)})),
 
+
+    lager:info("===testing using same wildcard certificate"),
+    rt:log_to_nodes([Node1, Node2], "wildcard certificates test"),
+    ?assertEqual(ok, test_connection({Node1, merge_config(SSLConfig8, BaseConf)},
+        {Node2, merge_config(SSLConfig8, BaseConf)})),
+
```

Comment:
Should have a test connecting wildcarded cert to non-wildcarded cert, without ACL, such as `SSLConfig3` <==> `SSLConfig8`